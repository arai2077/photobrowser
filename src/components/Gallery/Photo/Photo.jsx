// TODO:
// Figure out how to pass props here

import React from "react";
import { Link } from "react-router-dom";
import styled from "styled-components";

const StyledPhotoContainer = styled.div`
	display: flex;
	justify-content: center;
`;

const Photo = ({ url, title }) => (
	<StyledPhotoContainer>
		<img src={url} alt={title} />
		<h1>{title}</h1>
		<Link to="/">
			<button>Close</button>
		</Link>
	</StyledPhotoContainer>
);

export default Photo;
