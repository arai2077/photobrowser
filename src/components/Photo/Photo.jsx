import React from 'react';

const Photo = () => (
  <>
   Some photo
  </>
)

export default Photo;