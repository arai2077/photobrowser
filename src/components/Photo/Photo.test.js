import React from "react";
import { shallow } from "enzyme";
import Photo from "./Photo";

describe("Photo", () => {
  it("renders", () => {
    const component = shallow(<Photo />);

    expect(component).toMatchSnapshot();
  });
});
