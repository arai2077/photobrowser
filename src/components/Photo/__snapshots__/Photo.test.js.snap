// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo renders 1`] = `
<Fragment>
  Some photo
</Fragment>
`;
